baseline_collision.py

```diff
@@ -237,7 +235,7 @@ def run_collision_scene(json_path, controller, models, base_path="~/logs", num_s
         source_img = torch_to_np_img(images[-1])
 
         if step < 84:
-            controller.make_step_prediction(choice='plausible', confidence=1)
+            report[step] = make_step_prediction(choice="plausible", confidence=1)
```

Comment:
You had already fixed this in most places in your code, but you missed this one.